### PR TITLE
Issue 469 improve "or" error message

### PR DESCRIPTION
**Pull request by asolntsev**
## Proposed changes
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.
If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Checklist
- [x] Checkstyle and unit tests pass locally with my changes by running `gradle check chrome htmlunit` command
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: src/main/java/com/codeborne/selenide/conditions/Or.java [hunk 1]
```diff
@@ -18,26 +19,21 @@ public Or(String name, List<Condition> conditions) {
...
   @Override
   public String actualValue(Driver driver, WebElement element) {
-    return firstFailedCondition == null ? null : firstFailedCondition.actualValue(driver, element);
+    return conditions.stream().map(condition -> condition.actualValue(driver, element)).collect(joining(", "));
   }
 
   @Override
   public String toString() {
-    return firstFailedCondition == null ? super.toString() : firstFailedCondition.toString();
+    return getName() + ": " + conditions.stream().map(Condition::toString).collect(joining(" or "));
```
Comment (rosolko) [inline src/main/java/com/codeborne/selenide/conditions/Or.java]: It's better to introduce a new variable to complex stream operation and use string format for a final message

File: src/main/resources/versions.properties [hunk 2]
```diff
@@ -1,14 +1,17 @@
+# This file is copied from https://raw.githubusercontent.com/bonigarcia/webdrivermanager/master/src/main/resources/versions.properties
```
Comment (rosolko) [inline src/main/resources/versions.properties]: Maybe it's better to fetch this file from github url to use latest updates from source repository? And if it isn't available use local file instead?

File: src/test/java/com/codeborne/selenide/conditions/AttributeWithValueTest.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  }
+
+  @Test
+  void actualValue() {
+    assertThat(new AttributeWithValue("data-id", "expected").actualValue(driver, element)).isEqualTo("data-id=\"actual\"");
+  }
+
+  @Test
+  void tostring() {
+    assertThat(new AttributeWithValue("data-id", "expected").toString()).isEqualTo("attribute data-id=\"expected\"");
```
Comment (rosolko) [inline src/test/java/com/codeborne/selenide/conditions/AttributeWithValueTest.java]: Use `hasToString` assert

File: src/main/java/com/codeborne/selenide/conditions/Or.java (hunk 1 above)
Comment (asolntsev) [reply src/main/java/com/codeborne/selenide/conditions/Or.java]: done

**Pull request merged**